resources/js/provider.js

```diff
@@ -188,7 +188,12 @@
         if (!this._events[name]) {
           return
         }
-        this._events[name].forEach(cb => cb(data));
+        this._events[name].forEach(cb => {
+            try{
+                cb(data)
+            } catch(e){
+                setTimeout((()=>{throw e}))
+            }});
```

Comment:
Hi frank, Thank you very much for the PR.

I guess you are using `setTimeout` to throw the error just that it shouldn't stop the current js thread? Please can you explain, what is the benefit of doing that, can we just use an empty block here?